K299-Back/BackgroundService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using K299_Back.Controllers;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace K299_Back
+{
+    public class MyBackgroundService : IHostedService
+    {
+        private Timer _timer;
+        private int id = 1;
+        private readonly SolarDataController _otherController;
+
```

Comment:
Perkrovus serveri nepersistatys id vel į 1? Butu gerai arba isvalyt sugeneruotus irasus ir pradeti is naujo arba pasiimti paskutini id is jau sugeneruotos lenteles. @Qoldoon

Reply:
Jo, @Qoldoon dadek kad ieskotu is Simulated_inverter_record paskutinio iraso ir jeigu jis egzistuoja tada paimtu jo id + 1 ir nuo to ieskotu data. Jeigu nera recordu tada turetu buti 1